scripts/upgrade

```diff
@@ -23,6 +23,15 @@ ynh_systemd_action --service_name="$app" --action="stop" --log_path="systemd"
 #=================================================
 ynh_script_progression --message="Ensuring downward compatibility..." --weight=1
 
+# Fixes specific to packaging v2 migration
+if ynh_compare_current_package_version --comparison le --version 3.0.6.1196~ynh3
```

Comment:
It could just be 4.0 but yeah, fine with me.

Reply:
Ooof yes, I took the hard way. A `lt --version 4.0~ynh1` would have been equivalent and simpler to find.